src/app/components/ui/SubMenu/index.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Text from 'app/components/ui/Text';
+import DotsStepper from 'app/components/ui/DotsStepper';
+import PaddedContainer from 'app/components/ui/PaddedContainer';
+import { CenteredContainer, ActionsContainer } from './elements';
+
+
+const SubMenu = ( props ) => (
```

Comment:
En estos casos me parecce mejor 
```suggestion
const SubMenu = ({ activeStep, actions, data, time }) => (
```
Asi es mas facil leer que es lo que se esta reciviendo en las props  y que se esta usando